From 8481a8f65fec3b6b9bc0984f6ba36fbc82cc78ff Mon Sep 17 00:00:00 2001
From: Tomas Hozza <thozza@redhat.com>
Date: Thu, 18 Jul 2013 14:52:28 +0200
Subject: [PATCH 1/8] ldns-compare: add missing options into usage

Signed-off-by: Tomas Hozza <thozza@redhat.com>
---
 examples/ldns-compare-zones.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/examples/ldns-compare-zones.c b/examples/ldns-compare-zones.c
index 663b744..084b801 100644
--- a/examples/ldns-compare-zones.c
+++ b/examples/ldns-compare-zones.c
@@ -33,6 +33,8 @@ usage(char *prog)
 	printf("       -a - print all differences (-i -d -c)\n");
 	printf("       -s - do not exclude SOA record from comparison\n");
 	printf("       -z - do not sort zones\n");
+	printf("       -h - show usage and exit\n");
+	printf("       -v - show the version and exit\n");
 }
 
 int 
-- 
1.8.3.1


From 911a2bdcb03cbf36ac4591e1eea3c142b236350f Mon Sep 17 00:00:00 2001
From: Tomas Hozza <thozza@redhat.com>
Date: Thu, 18 Jul 2013 15:01:39 +0200
Subject: [PATCH 2/8] ldns-key2ds: Add -f option into man page

Signed-off-by: Tomas Hozza <thozza@redhat.com>
---
 examples/ldns-key2ds.1 | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/examples/ldns-key2ds.1 b/examples/ldns-key2ds.1
index 6fdbee5..a20ab96 100644
--- a/examples/ldns-key2ds.1
+++ b/examples/ldns-key2ds.1
@@ -17,6 +17,10 @@ SHA1 for RSASHA1, and so on.
 
 .SH OPTIONS
 .TP
+\fB-f\fR
+Ignore SEP flag (i.e. make DS records for any key)
+
+.TP
 \fB-n\fR
 Write the result DS Resource Record to stdout instead of a file
 
-- 
1.8.3.1


From 4d9ea09fac2965316381336e83e78a30cd9b0573 Mon Sep 17 00:00:00 2001
From: Tomas Hozza <thozza@redhat.com>
Date: Thu, 18 Jul 2013 15:07:51 +0200
Subject: [PATCH 3/8] ldns-keygen: remove unused options

Signed-off-by: Tomas Hozza <thozza@redhat.com>
---
 examples/ldns-keygen.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/examples/ldns-keygen.c b/examples/ldns-keygen.c
index 5765417..38ecfc1 100644
--- a/examples/ldns-keygen.c
+++ b/examples/ldns-keygen.c
@@ -71,7 +71,7 @@ main(int argc, char *argv[])
 	random = NULL;
 	ksk = false; /* don't create a ksk per default */
 
-	while ((c = getopt(argc, argv, "a:kb:r:v25")) != -1) {
+	while ((c = getopt(argc, argv, "a:kb:r:v")) != -1) {
 		switch (c) {
 		case 'a':
 			if (algorithm != 0) {
-- 
1.8.3.1


From 8726559f93ed85ae85b3667e4d8018c80714f3af Mon Sep 17 00:00:00 2001
From: Tomas Hozza <thozza@redhat.com>
Date: Thu, 18 Jul 2013 15:17:58 +0200
Subject: [PATCH 4/8] ldns-read-zone: document -b option in man page

Signed-off-by: Tomas Hozza <thozza@redhat.com>
---
 examples/ldns-read-zone.1 | 4 ++++
 examples/ldns-read-zone.c | 2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/examples/ldns-read-zone.1 b/examples/ldns-read-zone.1
index 7d4fd7d..e166658 100644
--- a/examples/ldns-read-zone.1
+++ b/examples/ldns-read-zone.1
@@ -22,6 +22,10 @@ that is not of type NSEC, NSEC3, RRSIG or DNSKEY. DS records are not
 printed.
 
 .TP
+\fB-b\fR
+Include Bubble Babble encoding of DS's.
+
+.TP
 \fB-0\fR
 Print a (null) for the RRSIG inception, expiry and key data. This option
 can be used when comparing different signing systems that use the same
diff --git a/examples/ldns-read-zone.c b/examples/ldns-read-zone.c
index efe187e..b0f5c3c 100644
--- a/examples/ldns-read-zone.c
+++ b/examples/ldns-read-zone.c
@@ -63,7 +63,7 @@ main(int argc, char **argv)
 				printf("Usage: %s [OPTIONS] <zonefile>\n", argv[0]);
 				printf("\tReads the zonefile and prints it.\n");
 				printf("\tThe RR count of the zone is printed to stderr.\n");
-				printf("\t-b include bubblebabble of DS's.\n");
+				printf("\t-b include Bubble Babble encoding of DS's.\n");
 				printf("\t-0 zeroize timestamps and signature in RRSIG records.\n");
 				printf("\t-c canonicalize all rrs in the zone.\n");
 				printf("\t-d only show DNSSEC data from the zone\n");
-- 
1.8.3.1


From 9292d3ffc9e929b50e23d6ccc71fe2201af11e25 Mon Sep 17 00:00:00 2001
From: Tomas Hozza <thozza@redhat.com>
Date: Thu, 18 Jul 2013 15:22:47 +0200
Subject: [PATCH 5/8] ldns-sigzone: remove unused -l option

Signed-off-by: Tomas Hozza <thozza@redhat.com>
---
 examples/ldns-signzone.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/examples/ldns-signzone.c b/examples/ldns-signzone.c
index 25ece3a..be505f4 100644
--- a/examples/ldns-signzone.c
+++ b/examples/ldns-signzone.c
@@ -379,7 +379,7 @@ main(int argc, char *argv[])
 
 	OPENSSL_config(NULL);
 
-	while ((c = getopt(argc, argv, "a:bde:f:i:k:lno:ps:t:vAE:K:")) != -1) {
+	while ((c = getopt(argc, argv, "a:bde:f:i:k:no:ps:t:vAE:K:")) != -1) {
 		switch (c) {
 		case 'a':
 			nsec3_algorithm = (uint8_t) atoi(optarg);
-- 
1.8.3.1


From 76a188d66f919b25d553c94e41781d6bf14704f1 Mon Sep 17 00:00:00 2001
From: Tomas Hozza <thozza@redhat.com>
Date: Thu, 18 Jul 2013 15:28:21 +0200
Subject: [PATCH 6/8] ldns-walk: add missing options into man page

Signed-off-by: Tomas Hozza <thozza@redhat.com>
---
 examples/ldns-walk.1 | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/examples/ldns-walk.1 b/examples/ldns-walk.1
index c8ace40..d4d1b6e 100644
--- a/examples/ldns-walk.1
+++ b/examples/ldns-walk.1
@@ -22,6 +22,14 @@ Of course the nameserver that is used must be DNSSEC-aware.
 
 .SH OPTIONS
 .TP
+\fB-4\fR
+Use only IPv4.
+
+.TP
+\fB-6\fR
+Use only IPv6.
+
+.TP
 \fB-f\fR
 Do a 'full' zone walk; by default, ldns-walk will only show the names, and types present at those names. If this option is given, all resource records will be printed.
 
@@ -31,6 +39,10 @@ Start the walk with this owner name. Useful when continuing the walk for a
 large zone.
 
 .TP
+\fB-v\fR \fIverbosity\fR
+Verbosity level [1-5].
+
+.TP
 \fB@\fR \fInameserver\fR
 Send the queries to this nameserver.
 
-- 
1.8.3.1


From 4317011bef2f58c045300ff7e479635d2e680809 Mon Sep 17 00:00:00 2001
From: Tomas Hozza <thozza@redhat.com>
Date: Thu, 18 Jul 2013 15:35:35 +0200
Subject: [PATCH 7/8] ldns-zcat: remove unused option -n and document option -v
 in man page

Signed-off-by: Tomas Hozza <thozza@redhat.com>
---
 examples/ldns-zcat.1 | 4 ++++
 examples/ldns-zcat.c | 2 +-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/examples/ldns-zcat.1 b/examples/ldns-zcat.1
index 9c4054d..4fd7ba6 100644
--- a/examples/ldns-zcat.1
+++ b/examples/ldns-zcat.1
@@ -18,6 +18,10 @@ The resulted zone file is printed to standard output.
 .B -o ORIGIN
 use ORIGIN when reading in the zone
 
+.TP
+.B -v
+show the version number and exit
+
 .SH AUTHOR
 Written by the ldns team as an example for ldns usage.
 
diff --git a/examples/ldns-zcat.c b/examples/ldns-zcat.c
index df4ecba..eea6ea7 100644
--- a/examples/ldns-zcat.c
+++ b/examples/ldns-zcat.c
@@ -54,7 +54,7 @@ main(int argc, char **argv)
 	progname = strdup(argv[0]);
 	origin = NULL;
 	
-	while ((c = getopt(argc, argv, "n:o:v")) != -1) {
+	while ((c = getopt(argc, argv, "o:v")) != -1) {
 		switch(c) {
 			case 'o':
 				origin = ldns_dname_new_frm_str(strdup(optarg));
-- 
1.8.3.1


From 147b12971f4308f6ebf1085fde5e61327c192ed5 Mon Sep 17 00:00:00 2001
From: Tomas Hozza <thozza@redhat.com>
Date: Thu, 18 Jul 2013 15:39:03 +0200
Subject: [PATCH 8/8] ldns-zsplit: document -v option in man page

Signed-off-by: Tomas Hozza <thozza@redhat.com>
---
 examples/ldns-zsplit.1 | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/examples/ldns-zsplit.1 b/examples/ldns-zsplit.1
index 8e64668..02dce40 100644
--- a/examples/ldns-zsplit.1
+++ b/examples/ldns-zsplit.1
@@ -27,6 +27,10 @@ use ORIGIN as origin when reading the zonefile.
 .B -z
 Sort the zone before splitting.
 
+.TP
+.B -v
+Show version number and exit.
+
 .SH AUTHOR
 Written by the ldns team as an example for ldns usage.
 
-- 
1.8.3.1

